Source/UI/NewUI/BillingForm/View/MainBillingFormTextField.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+final class MainBillingFormTextField: UITextField, BillingFormTextField {
+    var type: BillingFormCell?
+    
+    init(type: BillingFormCell?, tag: Int) {
+        self.type = type
+        super.init(frame: .zero)
+        self.tag = tag
```

Comment:
Why is the tag needed to be passed around? Is this just the standard `UIView.tag`? If so, can't it be set when needed by whoever needs it?

Reply:
It's used to define every textfield for every row